monitor-template.md

```diff
@@ -0,0 +1,188 @@
+#test in github
+
+INFO TAB
+
+\<monitor logo>
+# \<Monitor name>
+
+Metadata associated with the \<name> collectd plugin can be found _here_. The relevant code for the plugin can be found _here_.
+
+- [Description](#description)
+- [Features](#features)
+- [Use](#use)
+- [Metrics](#metrics)
+- [Dimensions](#dimensions)
+
+## Description
+
+Monitors \<what> using the information provided by \<what> which collects metrics from \<what> instances by hitting these endpoints: 
+* <link to item>
+* <link to item>
+
+## Features
+
+For more information on the source data, see <https://...>
```

Comment:
I'm not sure what this means. Would this follow a feature list and then link out to the originator of the monitor?

Plus the info below doesn't seem like Features to me. Maybe as headings under Usage....

Reply:
Took out the Features heading. The link is to more technical docs in the Integrations repo. It was part of the previous template for a subset of monitors.